src/export.rs

```diff
@@ -65,20 +66,23 @@ pub fn write_html_out(image_file_path: &str, color_data: &[Color], out_file_path
 /// ```
 ///
 pub fn write_json_out(color_data: &[Color], out_file_path: &str) -> std::io::Result<()> {
-    let mut file = File::create(format!("{}.json", out_file_path))?;
-    file.write_all(b"{")?;
-    file.write_all(b"\"palette\": [")?;
+    let mut json: String = String::new();
+    json.push_str("{");
```

Comment:
`clippy` is complaining here:

```
warning: calling `push_str()` using a single-character string literal
  --> src/export.rs:70:5
   |
70 |     json.push_str("{");
   |     ^^^^^^^^^^^^^^^^^^ help: consider using `push` with a character literal: `json.push('{')`

```

```suggestion
    json.push('{');
```